### DISCUSSION TITLE
26 packages are looking for funding

### DISCUSSION THREAD

**Discussion by mtjunqueira** [balta-io/forum#76] [Tira Dúvidas] [answered]
Estou tendo erro ao instalar o Angular na minha máquina como no curso. Sou dev c# e tenho visual studio na minha máquina.

c:\dev>npm install -g @angular/cli

changed 209 packages, and audited 210 packages in 44s

26 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

c:\dev>npm funds
Unknown command: "funds"

Did you mean this?
    npm fund # Retrieve funding information

To see a list of supported npm commands, run:
  npm help

c:\dev>npm fund @Angular/cli
npm ERR! code ENOFUND
npm ERR! No valid funding method available for: @Angular/cli

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\mtjun\AppData\Local\npm-cache\_logs\2022-11-01T20_07_16_892Z-debug-0.log

c:\dev>

c:\dev>

c:\dev>

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @mtjunqueira acredito que ele instalou corretamente, só não está rodando o audit... se você utilizar o comando `ng --version` verá a versão do angular instalada.
